### PR TITLE
Bump to ember-cli 2.12.0-beta.1

### PR DESCRIPTION
**Pull request by kratiahuja**
This is to allow to write test for the serve refactor (PR coming soon - TM).

cc: @rwjblue

### CODE REVIEW

**Comment by kratiahuja** [comment]
Sure will do. Surprisingly I didn't see that in the addon release diff.

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016
+Copyright (c) 2017
```
Comment (kellyselden) [inline LICENSE.md]: I wouldn't update copyright year.

Comment (kratiahuja) [inline LICENSE.md]: Any reason why?

Comment (kratiahuja) [inline LICENSE.md]: Any reason why?

Comment (rwjblue) [inline LICENSE.md]: Ya, we have fixed the blueprint to not touch this file in future updates.

@kratiahuja - Can you tweak to make it `2015-2017`?

Comment (rwjblue) [inline LICENSE.md]: @kratiahuja - See https://github.com/ember-cli/ember-cli/issues/6691

Comment (kratiahuja) [inline LICENSE.md]: Sure will do.

**Comment by kratiahuja** [comment]
Fixed per suggestion.

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**